### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by aku2nnya**
@MarissaShieh Please check my PR!!
https://trello.com/c/Q1CplUMY/53-2-create-initial-folder-file-setup-for-the-project

### CODE REVIEW

**Comment by MarissaShieh** [comment]
@aku2nnya consider changing bundle.js to app.js, just for uniformity with the other modules.
Otherwise, it looks good to me!

File: package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "carousel_module",
+  "version": "1.0.0",
+  "description": "Component to handle the main photo gallery of the checkout page.",
+  "main": "bundle.js",
```
Comment (MarissaShieh) [inline package.json]:
```suggestion
  "main": "app.js",
```

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            loader: 'babel-loader',
+            options: {
+              presets: ['@babel/preset-react', '@babel/preset-env']
+            }
+        }
+      }
+    ]
+  },
+  output: {
+    filename: 'bundle.js',
```
Comment (MarissaShieh) [inline webpack.config.js]:
```suggestion
    filename: 'app.js',
```

**Review by MarissaShieh — CHANGES_REQUESTED** [review]

**Comment by aku2nnya** [comment]
@MarissaShieh I finished making the changes, if you can please re-review that would be great. Thanks

**Comment by MarissaShieh** [comment]
@aku2nnya You're good to go!

**Review by MarissaShieh — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  },
+  "keywords": [],
+  "author": "Toshihiro Akabori",
+  "license": "",
+  "bugs": {
+    "url": "https://github.com/abode-ly/carousel_module/issues"
+  },
+  "homepage": "https://github.com/abode-ly/carousel_module#readme",
+  "dependencies": {
+    "bootstrap": "^4.3.1",
```
Comment (chinkevin25) [inline package.json:23]: Let's not use bootstrap! We should use CSS that we write. Bootstrap is great for prototyping but we want to learn how to do our own CSS

**Review by chinkevin25 — COMMENTED** [review]
Good PR! 

Let's make sure we write our own CSS and not use Bootstrap. Bootstrap is a really cool library and great for prototyping and worth learning but let's make sure we write our own CSS first.

File: package.json (hunk 3 above)
Comment (aku2nnya) [reply package.json:23]: @chinkevin25 
Yes, we've heard about this and have removed it from dependencies in the following PR.
Thanks.

**Pull request merged**